keanu-project/src/test/java/io/improbable/keanu/vertices/dbl/probabilistic/StudentTVertexTest.java

```diff
@@ -0,0 +1,190 @@
+package io.improbable.keanu.vertices.dbl.probabilistic;
+
+import io.improbable.keanu.vertices.dbltensor.DoubleTensor;
+import org.apache.commons.math3.distribution.TDistribution;
+import org.apache.commons.math3.stat.descriptive.SummaryStatistics;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static java.lang.Math.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+import static junit.framework.TestCase.assertEquals;
+
+/**
+ *
```

Comment:
Remove or add comments

Reply:
I believe this is done, please advise if not.